Задача 1. Даны две независимые выборки. Не соблюдается условие нормальности
x1 380,420,290
y1 140,360,200,900
Сделайте вывод по результатам, полученным с помощью функции, имеются ли статистические различия между группами?

Принимаем α=0,05.

In [3]:
import numpy as np
from scipy import  stats

In [4]:
x1 = np.array([380,420,290])
y1 = np.array([140,360,200,900])

In [5]:
stats.mannwhitneyu(x1, y1)

MannwhitneyuResult(statistic=8.0, pvalue=0.6285714285714286)

Т.к. pvalue=0.6285714285714286 > α=0,05,  можно говорить о том что статистически значимых различий в группах x1 b y1 нет.